src/App.tsx

```diff
@@ -43,6 +43,7 @@ const App = (props: Props) => {
       renderWidgetToggleButton={props.renderWidgetToggleButton}
       serviceName={props.serviceName}
       callbacks={props.callbacks}
+      fixateView={props.fixateView}
```

Comment:
I'm not sure, so I'm asking a question. Is the word 'fixate' commonly used?
If not, may I suggest using another familiar word to determine the mobile/desktop mode of the widget?

Reply:
How about putting 'isMobile(props.fixateView)' into a constant state and using it?
Using a function to handle it might lead to overlooked areas

Reply:
@bang9 please do so! I was not sure of the name either

Reply:
@bang9 ah good idea. We can add a prop to ConstantContextProps

Reply:
How about [deviceType](https://developer.chrome.com/docs/devtools/device-mode#device)?
<img width="596" alt="image" src="https://github.com/sendbird/chat-ai-widget/assets/26326015/a566983c-6fac-4a1a-b221-073230d091de">

Reply:
or isMobile?

Reply:
isMobile LGTM!

Reply:
let's change to deviceType because it isMobile implies dynamic. But what we meant is fixated view. So  deviceType is better name